tools/analysis/slither/docker-compose.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+  slither:
+    build: slither-analysis # WITH ecrecover DETECTOR preinstalled!
```

Comment:
q: are there steps anywhere to show how to preinstall the detector?

Reply:
Added it here:
https://github.com/hashgraph/hedera-smart-contracts/pull/740/files#diff-a0145f8b7b5f0359d5db591110cc968501542b54507461c7ccf21ffc6ee53367R117